locale/en-us/cancel.voc

```diff
@@ -0,0 +1,9 @@
+cancel
```

Comment:
This can be removed; the resource will be resolved from neon_core (or mycroft/ovos-core)